components/ScheduleSection.js

```diff
@@ -0,0 +1,242 @@
+import SectionTitle from './SectionTitle'
+
+const sessions = [
```

Comment:
In my opinion, we don’t need to create this array, because it adds a layer of indirection to our code, which makes our (already complex) schedule more complex.

Using `type: 'FLASH'` leads to if-check down below and leads to a very long component. Most components should not be longer than 50 lines.

__React gives us another way to compose code:__ By composing React elements, it helps us express our schedule more expressively.

For example, I would prefer that the schedule be expressed this way:

```js
function Schedule () {
  return (
    <ScheduleTable>
      <ActivityRow time='10:00 - 10:10'>Welcome Speech by Venue</ActivityRow>
      <SessionRow time='10:30 - 11:15'>
        <SessionItem
          topic='Let’s start with React in the right way'
          speakerName='Somkiat Puisungnoen'
        />
        <SessionItem
          topic='Scalable React app Architecture'
          speakerName='Zack Siri'
        />
      </SessionRow>
      ⋮
    </ScheduleTable>
  )
}
```